samples/javaconfig/saml2login/src/main/java/org/springframework/security/samples/config/SecurityConfig.java

```diff
@@ -40,27 +45,43 @@
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
 	RelyingPartyRegistration getSaml2AuthenticationConfiguration() throws Exception {
-		//remote IDP entity ID
+		// remote IDP entity ID
```

Comment:
Nothing wrong with reformatting as part of a PR. 
Would you be able to put that into a separate commit, one that doesn't change functionality, so that reviewers can focus on what actually changed?

Reply:
Hi Fillip,
I used the suggested formatter config for Eclipse, which is part of spring-security from the file `etc/eclipse/eclipse-code-formatter.xml`.
Reverting the automatic formatting for the files I changed would have been too much manual work, and I had many local test-builds that failed because of the formatting. So I decided to commit the files with all the automatic changes from the eclipse-code-formatter that went through the test.